tests/ui/variance/variance-associated-consts.rs

```diff
@@ -10,7 +10,7 @@ trait Trait {
 }
 
 #[rustc_variance]
-struct Foo<T: Trait> { //~ ERROR [o]
+struct Foo<T: Trait> { //~ ERROR E0208
```

Comment:
This is first a test tool for the compiler.
In this idea, losing the short version of the information makes the test much less useful.